addon/utils/load-polyfills.js

```diff
@@ -0,0 +1,32 @@
+export async function loadPolyfills() {
+  await Promise.all([
```

Comment:
just curious why we're using native Promise instead of RSVP here. is the plan to phase that libary out?

Reply:
No reason. I've been trying to remember to use it when I can since `RSVP` is eventually going to go away. This might be the first time I actually remembered. 😄